# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

UnloadTrayActionGoal action_goal
UnloadTrayActionResult action_result
UnloadTrayActionFeedback action_feedback
